phpunit.xml.dist

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/8.4/phpunit.xsd"
+         colors="true"
+         defaultTestSuite="unit"
+         bootstrap="vendor/autoload.php"
+>
+    <php>
+        <ini name="error_reporting" value="-1" />
+        <env name="SYMFONY_DEPRECATIONS_HELPER" value="max[self]=0" />
+        <env name="SYMFONY_PHPUNIT_VERSION" value="8.4" />
+        <env name="KERNEL_CLASS" value="Contao\CoreBundle\Tests\Functional\app\AppKernel" />
+        <env name="APP_SECRET" value="foobar" />
+        <env name="DATABASE_URL" value="mysql://root@localhost:3306/contao_test" />
+    </php>
+
+    <testsuites>
+        <testsuite name="unit">
+            <directory>./*-bundle/tests</directory>
+            <exclude>./*-bundle/tests/Fixtures</exclude>
+            <exclude>./*-bundle/tests/Functional</exclude>
+        </testsuite>
+        <testsuite name="functional">
+            <directory>./core-bundle/tests/Functional</directory>
+            <exclude>./core-bundle/tests/Functional/app</exclude>
+        </testsuite>
+        <testsuite name="coverage">
+            <directory>./*-bundle/tests</directory>
+            <exclude>./*-bundle/tests/Fixtures</exclude>
+            <exclude>./core-bundle/tests/Contao</exclude>
+            <exclude>./core-bundle/tests/Functional</exclude>
+        </testsuite>
+    </testsuites>
+
+    <filter>
+        <whitelist>
+            <directory>./calendar-bundle/src</directory>
+            <directory>./core-bundle/src</directory>
+            <directory>./faq-bundle/src</directory>
+            <directory>./manager-bundle/src</directory>
+            <directory>./news-bundle/src</directory>
+            <exclude>
+                <directory>./calendar-bundle/src/Resources</directory>
+                <directory>./core-bundle/src/Resources</directory>
+                <directory>./faq-bundle/src/Resources</directory>
+                <directory>./manager-bundle/src/Resources</directory>
+                <directory>./news-bundle/src/Resources</directory>
+                <!-- exclude files with symbols and side-effects -->
```

Comment:
What are does?

Reply:
I have just copied this from the existing `core-bundle/phpunit.xml.dist` file:

https://github.com/contao/contao/blob/198adf77b8d3d50dd8a664062814bdce216d9025/core-bundle/phpunit.xml.dist#L42-L45

It excludes classes with symbols and side-effects from the coverage report.